app/views/projects/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> <%= @project.name %> </h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
This is another opportunity to add a model method. This is technically database logic that should live in the model.